src/demo-app/chips/chips-demo.ts

```diff
@@ -47,6 +52,16 @@ export class ChipsDemo {
     }
   }
 
+  remove(person: Person): void {
+    if (person) {
```

Comment:
I think this check can be dropped, because TypeScript already validates the person is *not* undefined.

Also the `indexOf` would just return `-1` if it's really undefined.